build/nv.d3.js

```diff
@@ -14179,13 +14180,17 @@ nv.models.sunburst = function() {
                                 return 0;
                             }
                         })
-                        .attr("transform", function() {
+                        .attr("transform", function(d1) {
                             var width = this.getBBox().width;
-                            if(e.depth === 0)
-                            return "translate(" + (width / 2 * - 1) + ",0)";
-                            else if(e.depth === d.depth){
+                            if(e.depth === 0) {
+                                return "translate(" + (width / 2 * - 1) + ",0)";
```

Comment:
This condition applies to the first text node on the tree, which is the one on the centre of the sunburst chart. I have noticed when enlarging the `font-size` I am noticing this is not behaving as expected. See image below. Will need to do a small adjustment...

<img src="https://cloud.githubusercontent.com/assets/17259166/24638903/f160201e-18a0-11e7-9682-d6b272e6a9c0.png" width="300">